docs/source/suite.rst

```diff
@@ -104,10 +182,11 @@ The 3D dexterous human arm reaching and manipulation.
 
 This musculoskeletal model is comprised of 27 Degree of Freedom, and 63 muscles-tendon units.
 
-.. image:: images/myoArm.png
-  :height: 200
+Model specification can be found on our github repo of `myoArm <https://github.com/MyoHub/myo_sim/tree/dbaecde82a18341661d4eb6d41928f26dc017bdd/arm>`__
```

Comment:
Edit it to point to main of myoSim arm. Just checked the error occurs because the default linking from myosuite to myoSim is in a branch called "dbaecde"

Reply:
updated to point to main